lib/modules/datasource/conan/index.spec.ts

```diff
@@ -383,5 +383,94 @@ describe('modules/datasource/conan/index', () => {
         })
       ).toBeNull();
     });
+
```

Comment:
This PR does not yet automatically close the relevant issue?

Correct syntax is: `Fixes #23373`